java/Nikiforov-Oleksandr/src/com/zoolatech/lecture1/tasks/_1/Task1.java

```diff
@@ -0,0 +1,20 @@
+package com.zoolatech.lecture1.tasks._1;
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts numbers of miles as a floating point number,
+ * converts it to kilometers and outputs the result.
+ */
+
+
+
+public class Task1 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        milesToKm(sc.nextFloat());
+    }
+
+    public static void milesToKm(float distance){
+        System.out.println("Distance in km: "+ distance * 1.60934412f);
```

Comment:
Could you please define a variable for this operation? It might make the code more readable